go/v1beta1/storage/filtering/filtering.go

```diff
@@ -34,6 +35,8 @@ func NewFilterer() Filterer {
 	return &filterer{}
 }
 
+var elasticsearchSpecialCharacterRegex = regexp.MustCompile(`([\-=&|!(){}\[\]^"~*?:\\/])`)
```

Comment:
Good foresight!! I never would have even thought of this until it broke 

That covers everything listed here except for the < > (which I know are unescapable) and the "+" :

> The reserved characters are: + - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /

Also do we need to do anything with spaces?

Reply:
for some reason, cel eats the `+` when parsing the filter expression.  so we didn't include it here.  we shouldn't have to do anything with spaces